src/rom_header_gf.c

```diff
@@ -151,8 +151,8 @@ static const struct GFRomHeader sGFRomHeader = {
     .externalEventDataOffset = offsetof(struct SaveBlock1, externalEventData),
     .unk18 = 0x00000000,
     .speciesInfo = gSpeciesInfo,
-    .abilityNames = gAbilityNames,
-    .abilityDescriptions = gAbilityDescriptionPointers,
+    //.abilityNames = gAbilityNames, //handled in gAbilities
+    //.abilityDescriptions = gAbilityDescriptionPointers, //handled in gAbilities
```

Comment:
I think it may be worth adding a pointer to `gAbilities` in the RHH ROM header so that external tools have some way to find the names/descriptions of abilities (and possibly `ABILITY_COUNT` too, ala #3831)?

Reply:
Added the struct pointer to the RHH header!